bin/schema_generator.sh

```diff
@@ -87,7 +86,28 @@ function filter_schemas() {
     find . -name '*.bq' | grep -f $DISALLOWLIST | xargs rm -f
 }
 
-function commit_schemas() {
+function create_changelog() {
+    # Generate output for referencing to the changeset in the source branch.
+    # Useful references for --format and --pretty
+    # https://stackoverflow.com/questions/25563455/how-do-i-get-last-commit-date-from-git-repository
+    # https://stackoverflow.com/questions/1441010/the-shortest-possible-output-from-git-log-containing-author-and-date
+    # https://git-scm.com/docs/git-log/1.8.0#git-log---daterelativelocaldefaultisorfcshortraw
+    local start_date
+    # NOTES: Since this function is looking at commit dates (such as rebases),
+    # it's best to enforce squash/rebase commits onto the master branch. If this
+    # isn't enforced, it's possible to miss out on certain commits to the
+    # generated branch. Another solution is to generate a tag on master before
+    # the generator is run. However, the heuristic of using the latest commit
+    # date works well enough.
+    start_date=$(git log "${MPS_BRANCH_PUBLISH}" -1 --format=%cd --date=iso)
+    git log "${MPS_BRANCH_SOURCE}" \
+        --since="$start_date" \
+        --pretty=format:"%h%x09%cd%x09%s" \
+        --date=iso \
+        | cat
```

Comment:
`git log` writes to a pager by default (e.g. less), so piping to cat allows this to be used as input elsewhere.

Reply:
git has a `--no-pager` flag as well that you should probably leverage

Reply:
After playing around with the option, it looks like there wasn't actually a problem with paging. I reran the script without the pipe into cat, and it looks to be working correctly.